2025-07-15 22:22:26,169 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_aU6DIODwxNlFRxrY3WipBPjz "HTTP/1.1 200 OK"
2025-07-15 22:22:26,808 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_1dQLsAz9p6T2cQyGtnjSeXnv "HTTP/1.1 200 OK"
2025-07-15 22:22:27,419 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_II6EhFFETHGmydpVxGrOVzQG "HTTP/1.1 200 OK"
2025-07-15 22:22:28,095 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_nmFEiTZ2mE7XlUVlUv5PXAzu "HTTP/1.1 200 OK"
2025-07-15 22:22:28,830 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_jtOdIxiHK1UsVkXaCxM8y0PS "HTTP/1.1 200 OK"
2025-07-15 22:22:29,434 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_cErO4m6RZdfHQPAT3wVagp2z "HTTP/1.1 200 OK"
2025-07-15 22:22:29,993 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_nJzNpbdII7UzbOGiiSFcu09u "HTTP/1.1 200 OK"
2025-07-15 22:22:30,476 - INFO - HTTP Request: GET https://api.openai.com/v1/assistants/asst_FM5jrNCeRHxy3MpMueV1RkED "HTTP/1.1 200 OK"
2025-07-15 22:22:30,482 - INFO - Starting bot...
2025-07-15 22:22:42,719 - INFO - HTTP Request: POST https://api.openai.com/v1/threads "HTTP/1.1 200 OK"
2025-07-15 22:22:42,739 - INFO - Thread was created with id thread_EJR5oS41KFZTkPnBYUx5GvxX
2025-07-15 22:22:57,628 - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/messages "HTTP/1.1 200 OK"
2025-07-15 22:23:03,970 - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs "HTTP/1.1 200 OK"
2025-07-15 22:23:05,395 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_zSHIhKWSWCYAmVrQKqodRUFo "HTTP/1.1 200 OK"
2025-07-15 22:23:06,804 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_zSHIhKWSWCYAmVrQKqodRUFo "HTTP/1.1 200 OK"
2025-07-15 22:23:08,359 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_zSHIhKWSWCYAmVrQKqodRUFo "HTTP/1.1 200 OK"
2025-07-15 22:23:08,382 - INFO - ORCHESTRATOR RUN COMPLETED:
Input tokens: 1269 (0.0031725000000000004$)
Input tokens: 49 (0.0004900000000000001$)
2025-07-15 22:23:08,726 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/messages?limit=1 "HTTP/1.1 200 OK"
2025-07-15 22:23:42,449 - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs "HTTP/1.1 200 OK"
2025-07-15 22:23:43,871 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_kduPSjzRRwlZhVatittagZmV "HTTP/1.1 200 OK"
2025-07-15 22:23:45,254 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_kduPSjzRRwlZhVatittagZmV "HTTP/1.1 200 OK"
2025-07-15 22:23:46,828 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_kduPSjzRRwlZhVatittagZmV "HTTP/1.1 200 OK"
2025-07-15 22:23:48,189 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_kduPSjzRRwlZhVatittagZmV "HTTP/1.1 200 OK"
2025-07-15 22:23:49,643 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_kduPSjzRRwlZhVatittagZmV "HTTP/1.1 200 OK"
2025-07-15 22:23:51,021 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_kduPSjzRRwlZhVatittagZmV "HTTP/1.1 200 OK"
2025-07-15 22:23:51,345 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/messages?limit=1 "HTTP/1.1 200 OK"
2025-07-15 22:23:51,576 - INFO - HTTP Request: GET https://api.openai.com/v1/files/file-KsAVnNcsT5KdgGsBYqxTah "HTTP/1.1 200 OK"
2025-07-15 22:23:51,579 - WARNING - Failed find source in pdf_mapping.json. 'MS900_UM_ENG'
Traceback (most recent call last):
  File "D:\Projects\open-ai-assistant-test\orchestrator_for_tg.py", line 564, in route_to_assistant
    source_file_path = pdf_mapping[name_without_ext]
                       ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
KeyError: 'MS900_UM_ENG'
2025-07-15 22:26:55,956 - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/messages "HTTP/1.1 200 OK"
2025-07-15 22:26:57,254 - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs "HTTP/1.1 200 OK"
2025-07-15 22:26:58,691 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_Brk7jwHSF3n09AT1EeF5E0cu "HTTP/1.1 200 OK"
2025-07-15 22:27:00,075 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_Brk7jwHSF3n09AT1EeF5E0cu "HTTP/1.1 200 OK"
2025-07-15 22:27:00,078 - INFO - ORCHESTRATOR RUN COMPLETED:
Input tokens: 1676 (0.00419$)
Input tokens: 49 (0.0004900000000000001$)
2025-07-15 22:27:00,306 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/messages?limit=1 "HTTP/1.1 200 OK"
2025-07-15 22:27:26,573 - INFO - HTTP Request: POST https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs "HTTP/1.1 200 OK"
2025-07-15 22:27:28,110 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:29,506 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:30,875 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:34,362 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:35,925 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:37,276 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:38,683 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:40,147 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:41,834 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/runs/run_W3iQnw3z7PBI7FGRjCIpYRMh "HTTP/1.1 200 OK"
2025-07-15 22:27:43,799 - INFO - HTTP Request: GET https://api.openai.com/v1/threads/thread_EJR5oS41KFZTkPnBYUx5GvxX/messages?limit=1 "HTTP/1.1 200 OK"
2025-07-15 22:27:44,092 - INFO - HTTP Request: GET https://api.openai.com/v1/files/file-Es6Mi9gCKgRZRYvayYYz7D "HTTP/1.1 200 OK"
2025-07-15 22:27:44,095 - WARNING - Failed find source in pdf_mapping.json. 'comp_table_test_benches_starter_generator'
Traceback (most recent call last):
  File "D:\Projects\open-ai-assistant-test\orchestrator_for_tg.py", line 564, in route_to_assistant
    source_file_path = pdf_mapping[name_without_ext]
                       ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
KeyError: 'comp_table_test_benches_starter_generator'
